static/panes/diff.ts

```diff
@@ -156,9 +156,9 @@ export class Diff extends MonacoPane<monaco.editor.IStandaloneDiffEditor, DiffSt
                     {id: DiffType.ExecStdErr, name: 'Execution stderr'},
                     {id: DiffType.GNAT_ExpandedCode, name: 'GNAT Expanded Code'},
                     {id: DiffType.GNAT_Tree, name: 'GNAT Tree Code'},
-                ],
+                ] as any[],
```

Comment:
this (and others) is presumably to fix some weird TomSelect things?

I've filed a couple issues:
- orchidjs/tom-select#500
- orchidjs/tom-select#501

which aim to fix this; in theory they don't affect us unless we upgrade tom-select to 2.2.x though (maybe you did an `npm update` and reverted it later? I do'nt see that there's a `package.json` or lockfile change here?)